### DISCUSSION TITLE
Welcome to MyMood Discussions!

### DISCUSSION THREAD

**Discussion by NimaKhajehpour** [NimaKhajehpour/MyMood#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]


**Reply by NimaKhajehpour** [reply]
hey @gnu1dea thanks for using the app i always feel so happy to see people say that they use my app 😁
thanks for the feedback and opening the thread here.
i need a popular platform in which we can be more in touch with the community because many users dont have (git)hub or they just dont check it very often. so thats why im thinking of a good platform to optimistically include all or hopefully most of the users there so communication is easier for everyone.
i made telegram i thought would be a good start but then i realized it is not that much used or at least people dont feel like joining in communities that much.
so i thought of one of the best social medias to make a community which is discord.
but i guess it is not going to be ideal and i dont know any other good socials to make a community for my apps specially mymood.
i could make my own community app but some users dont like making api calls or giving the apps internet access at any time. so i really dont know what to do about it. feel free to give me any ideas for that, i dont know how matrix is used for communities and how many people use it.
i do appreciate the feedback and use of my app, i am revamping everything by the way so stay tuned for that 😉
i really like when users come forward and talk about anything related to my apps.
thanks a lot
